.github/actions/publish-image/action.yml

```diff
@@ -0,0 +1,57 @@
+name: "Publish Image to domain"
```

Comment:
can we have it just "Publish Image" to avoid confusions on what "domain" is

Reply:
or better "Publish Docker Images"

Reply:
Changed